library_generation/utils/utilities.sh

```diff
@@ -149,16 +136,17 @@ get_protoc_version() {
 # DOCKER_GRPC_VERSION respectively, this function will instead set "protoc_path"
 # and "grpc_path" to DOCKER_PROTOC_PATH and DOCKER_GRPC_PATH respectively (no
 # download), since the docker image will have downloaded these tools beforehand.
+#
 # For the case of gapic-generator-java, no env var will be exported for the
 # upstream flow, but instead it will be assigned a default filename that will be
-# referenced by the file `library_generation/gapic-generator-java-wrapper`.
+# referenced by the file `library_generation/gapic-generator-java-wrapper`. Note
+# that we assume an env var DOCKER_GAPIC_GENERATOR_LOCATION to point to the jar
```

Comment:
Could you remind me where to set `DOCKER_GAPIC_GENERATOR_LOCATION`?

I didn't find it in the development guide.

Reply:
Ah, this is wrong. I updated the comment to explain about the well-known location. Thanks for the catch!